include/rocksdb/env.h

```diff
@@ -473,6 +474,12 @@ class RandomAccessFile {
   virtual Status Read(uint64_t offset, size_t n, Slice* result,
                       char* scratch) const = 0;
 
+  // Readahead from the file backwards from offset - 1 by n bytes for caching.
+  // Implementation details are up to subclasses
+  virtual Status Prefetch(uint64_t offset, size_t n) {
```

Comment:
Prefetch should be from [offset, offset + n], instead of backward, just like what does Read() does.